tools/infrasys/src/error.rs

```diff
@@ -149,17 +156,27 @@ pub enum Error {
     ))]
     PutPolicy {
         bucket_name: String,
-        source: rusoto_core::RusotoError<rusoto_s3::PutBucketPolicyError>,
+        source: aws_sdk_s3::types::SdkError<aws_sdk_s3::error::PutBucketPolicyError>,
     },
 
     #[snafu(display("Failed to create async runtime: {}", source))]
     Runtime { source: std::io::Error },
 
+    /// The library failed to join 'tokio Runtime'.
+    #[snafu(display("Unable to join tokio thread used to offload async workloads"))]
+    ThreadJoin,
+
     #[snafu(display("'tuftool {}' returned {}", command, code))]
     TuftoolResult { command: String, code: String },
 
     #[snafu(display("Failed to start tuftool: {}", source))]
     TuftoolSpawn { source: io::Error },
+
+    #[snafu(display("Unable to create tokio runtime: {}", source))]
+    RuntimeCreation {
+        source: std::io::Error,
+        backtrace: Backtrace,
```

Comment:
In [pubsys](https://github.com/bottlerocket-os/bottlerocket/pull/2415) we ended up removing backtrace from RuntimeCreation.

Reply:
Or maybe drop this in favor of `Runtime` from L162-L163.